.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": ".venv/bin/python3.7"
```

Comment:
As this is ide settings, this is nbd but you probably want python3.6.

Reply:
I removed it and added to .gitignore.